funsor/terms.py

```diff
@@ -36,38 +37,22 @@ def subs_interpreter(cls, *args):
         return interpreter.reinterpret(expr)
 
 
-def alpha_substitute(expr, alpha_subs):
-
-    new_values = []
-    for v in expr._ast_values:
-        v = substitute(v, alpha_subs)
-        if isinstance(v, str) and v not in expr.fresh:
-            v = alpha_subs.get(v, v)
```

Comment:
Is this the only piece of logic that actually changed here or in #226 ?

Reply:
Yes, you could see this PR as 1. changing the logic for alpha converting non-Funsors; and 2. (unrelated to this line) moving the `to_funsor()` logic around. #226 additionally changes the `Independent` signature.

Reply:
I mean lines 44 and 45 specifically - it looks like everything else is the same, just duplicated?